src/main/resources/static/static/js/updateOrder.js

```diff
@@ -0,0 +1,70 @@
+//计算商品总价
+function updateFinalPrice() {
+    var cartTableElem = $("#itemTable").find("tr[id]");
+    var totalPrice = 0;
+    cartTableElem.each(function(cIndex, cElement) {
+        if($(cElement).find("#sItemChecked").is(":checked")){
+            var cNum = $(cElement).find(".ItemNo").val(); //商品数量
+            if(cNum < 1){
+                alert("请修改商品数量！");
+            }else{
+                var singlePrice = cNum * $(cElement).find(".sPrice").text(); //商品小计
+                totalPrice += singlePrice; //总价
+            }
+        }
+    });
+    $("#finalPrice").text(totalPrice);
+}
+//提交订单
+function submitOrder() {
+
+}
+//更新购物车
+function updateCart() {
+    var cartTableElem = $("#itemTable").find("tr[id]");
+    cartTableElem.each(function(cIndex, cElement) {
+        if($(cElement).find("#sItemChecked").is(":checked")){
+            var cElemId = $(cElement).find("#sItemID").val(); //商品ID
+            var cElemNum = $(cElement).find("#sItemNo").val(); //商品数量
+            if(cElemNum < 1){
+                alert("请修改商品数量！");    // check number on the backend
```

Comment:
Note: The man who write backend for this part, please use the backend to check if the number is valid and the item id is only an unsigned int.